platform-sdk/swirlds-platform-core/build.gradle.kts

```diff
@@ -15,41 +15,43 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("org.gradle.java-test-fixtures")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
+    id("com.swirlds.platform.maven-publish")
+    id("org.gradle.java-test-fixtures")
 }
 
 extraJavaModuleInfo { failOnMissingModuleInfo.set(false) }
 
 dependencies {
-  // Individual Dependencies
-  implementation(project(":swirlds-base"))
-  api(project(":swirlds-fchashmap"))
-  api(project(":swirlds-fcqueue"))
-  api(project(":swirlds-jasperdb"))
-  api(project(":swirlds-cli"))
-  api(project(":swirlds-base"))
-  compileOnly(libs.spotbugs.annotations)
-  runtimeOnly(project(":swirlds-config-impl"))
+    // Individual Dependencies
+    implementation(project(":swirlds-base"))
+    api(project(":swirlds-fchashmap"))
+    api(project(":swirlds-fcqueue"))
+    api(project(":swirlds-jasperdb"))
+    api(project(":swirlds-cli"))
+    api(project(":swirlds-base"))
+    testImplementation("org.jetbrains:annotations:23.0.0")
+    compileOnly(libs.spotbugs.annotations)
+    testCompileOnly(libs.spotbugs.annotations)
```

Comment:
This should go down in the `Test Dependencies` section (applies to all `build.gradle` files in this PR)

Reply:
moved

Reply:
ah!

Reply:
should be fixed in the 3 other locations now.  sorry for not fully reading your comment.

Reply:
No worries 👍